apps/api/models.py

```diff
@@ -102,24 +105,50 @@ class APIKey(models.Model):
     class Meta:
         db_table = 'api_key'
 
+    def __unicode__(self):
+        return (
+            u'<{cls} user={user}, type={type}, key={key} secret=...>'
+            .format(cls=self.__class__.__name__, key=self.key,
+                    type=self.type, user=self.user))
+
     @classmethod
     def get_jwt_key(cls, **query):
```

Comment:
what would the intention for that be? To prevent a typo when someone is using `get_jwt_key`? Because originally Andy suggested that it would prevent someone from accidentally passing through all user's keys when using this method. I don't see how requiring a `key` or a `user` would prevent that. As is, if you call `get_jwt_key` without a user or a key you will receive an error, one of 'multiple objects returned' or 'object does not exist'.